bot/utils/voice_recognition.py

```diff
@@ -0,0 +1,101 @@
+import os
+from google.cloud import speech
+import requests
+import logging
+import json
+from tempfile import NamedTemporaryFile
+import subprocess
+
+TOKEN = os.environ["TOKEN"]
+
+LANG = "ru-RU"
+SAMPLE_RATE_HERTZ = 48000
+ENCODING_TYPE = "LINEAR16"
+CHANNEL_COUNT = 1
+ENABLE_AUTOMATIC_PUNCTUATION = True
+
+logger = logging.getLogger(__name__)
+
+speechClient = speech.SpeechClient()
+
+
+def _get_audio_file_data(file_id):
+    """
+    gets audio file 
+    """
+    url = f"https://api.telegram.org/bot{TOKEN}/getFile?file_id={file_id}"
+    response = requests.get(url)
+    jsonResponse = response.json()
+    logger.info(json.dumps(jsonResponse, indent=4))
```

Comment:
Logs shouldn't be printed without context

Reply:
Fixed.